main.js

```diff
@@ -0,0 +1,77 @@
+
+var board = {
+
+  state: [[5, 4, 3, 2, 1],[],[]],
+
+  moveDisc: function(move,target){
```

Comment:
this function is doing too much, needs to be extracted into other functions like isWinner or isValidMovement